kube-prereq.adoc

```diff
@@ -8,28 +8,26 @@
 ////
 == Prerequisites
 
-Before you begin, have the following tools installed:
+Before you begin, have following tools must be installed:
```

Comment:
The updated sentence seems weird.  @Charlotte-Holt, is this the suggested change?

Reply:
Thank you for pointing it out. It's fixed.